src/vcpkg/build.cpp

```diff
@@ -309,7 +309,26 @@ namespace vcpkg::Build
         "",
         "Error: in triplet {triplet}: Unable to find a valid toolchain combination.\n    The requested target "
         "architecture was {value}\n    "
-        "The selected Visual Studio instance is at {path}\n    The available toolchain combinations are {list}\n");
+        "The selected Visual Studio instance is at {path}\n    The available toolchain combinations are {list}\n"
+        "Please check the required Visual Studio Components:\n"
+        "C++:\n"
+        "   Windows Universal C Runtime\n"
+        "   C++ core desktop features\n"
+        "   One of the following Visual Studio Build tools:\n"
+        "       * Microsoft.VisualStudio.Component.VC.140 (for Visual Studio 2015)\n"
```

Comment:
Are these component names localized in the VS Installer?

I think instead of having a gigantic block of text, it would be better to break it up into logical pieces that can be individually localized (list headings, component names).

Reply:
Can you provide a  sample please?